samples/cpp/tutorial_code/video/optical_flow/optical_flow.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/videoio.hpp>
+#include <opencv2/video.hpp>
+
+using namespace cv;
+using namespace std;
+
+int main()
+{
+    VideoCapture capture("slow_traffic_small.mp4");
```

Comment:
Where is located this file?

Reply:
This file which is also named "slow.flv" in the [python tutorial](https://github.com/opencv/opencv/blob/master/doc/py_tutorials/py_video/py_lucas_kanade/py_lucas_kanade.markdown) has actually been requested/missing multiple times in the forum, e.g. [here](http://answers.opencv.org/question/43986/python-tutorial-sample-source-file/), [here](http://answers.opencv.org/question/70166/download-example-file/) or [here](http://answers.opencv.org/question/102258/where-to-get-slowflv/). How do we deal with this?
PS: Is there a reason the opencv buildbot seems to be stuck at "Waiting for builds"?

Reply:
Agree.

Can we pass video via command line parameter?
We can show URL link for proposed video via "usage help". [Example](https://github.com/opencv/opencv/blob/4.1.0/samples/dnn/colorization.cpp#L51-L53).